assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -248,6 +250,7 @@ const baseActions = {
 		SettingsEditComponent,
 		SettingsViewComponent,
 		SetupComponent,
+		SettingsSetupIncompleteComponent = DefaultSettingsSetupIncomplete,
```

Comment:
This assignment belongs in the [`moduleDefaults`](https://github.com/google/site-kit-wp/blob/8c37f05d1f1a951f91292de6ec52209a3a82a956/assets/js/googlesitekit/modules/datastore/modules.js#L54) variable above, not as a default to the action.